iis-common/src/test/java/eu/dnetlib/iis/common/spark/avro/SparkAvroSaverTest.java

```diff
@@ -0,0 +1,199 @@
+package eu.dnetlib.iis.common.spark.avro;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+
+import org.apache.avro.Schema;
+import org.apache.avro.mapred.AvroKey;
+import org.apache.avro.mapreduce.AvroKeyOutputFormat;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.io.NullWritable;
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.function.PairFunction;
+import org.apache.spark.sql.DataFrame;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import scala.Tuple2;
+import eu.dnetlib.iis.IntegrationTest;
+
+
+/**
+ * @author Łukasz Dumiszewski
+ */
+@RunWith(MockitoJUnitRunner.class)
+@Category(IntegrationTest.class)
+public class SparkAvroSaverTest {
```

Comment:
This looks like unit test. Is there any reason that you set it as integration test?